File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.py
n = int(input())
for i in range(n):
    s = input()
    score = 0
    add = 1
    for i in range(len(s)):
        if s[i] == 'O':
            score += add
            add += 1
        else:
            add = 1
    print(score)

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.c
#include <stdio.h>
#include <string.h>
main() {
	int n;
	char arr[81];
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		int score = 0, add = 1;
		scanf("%s", arr);
		for (int j = 0; j < strlen(arr); j++) {
			if (arr[j] == 'O') {
				score += add;
				add++;
			}
			else
				add = 1;
		}
		printf("%d\n", score);
	}
}